content/get-started.md

```diff
@@ -2,6 +2,18 @@
 title: Get Started with Webpack
 ---
 
-* basic concepts, get webpack up and running, technical focus
-* target - the user should be able to complete this in 10 minutes
+`webpack` is a module bundler.A bundler is a tool that builds all your project files to a single asset file that can be loaded onto a browser.It requires that your application files be written as [modules](/concepts/modules).
+The project structure consists of a single entry file with references to other modules as dependencies creating a tree like dependency graph.The dependency modules can belong to your application or can be third party libraries.
```

Comment:
I'd much rather make this about webpack again, what about somethingn like "Webpack starts with your entry file, which has references to other modules, which have references to other modules, etc. This creates a tree-like _dependency graph_ that webpack can work with."